test/mapmatch.cc

```diff
@@ -170,9 +171,10 @@ void test_matcher() {
     // get the edges along that route shape
     boost::property_tree::ptree walked;
     std::string walked_json;
+
     try {
       walked_json = actor.trace_attributes(
-          R"({"costing":"auto","shape_match":"edge_walk","encoded_polyline":")" +
+          R"({"date_time":{"type":1,"value":"2019-10-31T18:30"},"costing":"auto","shape_match":"edge_walk","encoded_polyline":")" +
```

Comment:
for every test we first do an edge walk match with a date time on it to check that that code path works

Reply:
and then for the regular map match we are using the to_locations function above to set timestamps on every resampled noisy gps point to trigger timestamp matching in that path way.